go/vt/topo/shard.go

```diff
@@ -257,7 +258,7 @@ func (ts *Server) UpdateShardFields(ctx context.Context, keyspace, shard string,
 // CreateShard creates a new shard and tries to fill in the right information.
 // This will lock the Keyspace, as we may be looking at other shard servedTypes.
 // Using GetOrCreateShard is probably a better idea for most use cases.
-func (ts *Server) CreateShard(ctx context.Context, keyspace, shard string) (err error) {
+func (ts *Server) CreateShard(ctx context.Context, keyspace, shard string, vtorcConfig *vtorcdatapb.Shard) (err error) {
```

Comment:
Why do we necessarily need to apply a vtorc config on shard creation? Shouldn't we always create with default "false" (or the equivalent of 'no specific action in place'?)

What is the scenario for creating a shard with specific shard-disable-ers instruction?

Reply:
@shlomi-noach good question, so far we haven't used this

I added it for hypothetical flexibility if someone wanted to create a disabled shard, but they could just call the RPC post-creation to achieve the same thing

In hindsight, I think I'll remove this for now and wait for it to have a use case justified 👍

Reply:
Thank you Tim. I agree, let;s remove for now and wait for a use case.